kanopi/rooseveltedu
# 26124955: Template - Footer

## User Story
As a user of the Roosevelt site, I need an easy and intuitive way to navigate, have quick access to important sections, social media links, information regarding locations, privacy policy information and any other important information regarding the site, plus have the same look and feel through the website, so I can quickly find what I’m looking for.

## Acceptance Criteria
 - [x] Footer quick links section should be available - space for three (Desktop & Mobile)
 - [x] Locations info section should be available (Desktop & Mobile)
 - [x] Footer menu - only one level (Desktop & Mobile)
 - [x] Roosevelt motto section should be available (Desktop & Mobile)
 - [x] Social Media links should be available (Desktop & Mobile)
 - [x] Copyright and footer utility menu section should be on the footer (Desktop & Mobile)
 - [x] Footer has to match the design (Desktop & Mobile)
 - [x] Accessibility check

## Wireframes
[Zeplin file](https://zpl.io/bL0RY5d)

## Design
[Roosevelt Redesign](https://content.roosevelt.edu/redesign.htm)
_note: you will need VPN access_

## Technical Details
Mobile stacks following the order of the desktop but there are some slightly changes in design as not displaying the ‘R’ image for the motto.

## Affected URL
[Test environment](https://pr-73-rooseveltedu.pantheonsite.io/)

## Related Tickets
* [Template - Footer - Desktop - Footer Quicklinks](https://kanopi.teamwork.com/#/tasks/26124957)
* [Template - Footer - Mobile - Footer Quicklinks](https://kanopi.teamwork.com/#/tasks/26124958)
* [Template - Footer - Desktop - Locations](https://kanopi.teamwork.com/#/tasks/26124959)
* [Template - Footer - Mobile - Locations](https://kanopi.teamwork.com/#/tasks/26124960)
* [Template - Footer - Desktop - Footer Menu](https://kanopi.teamwork.com/#/tasks/26124961)
* [Template - Footer - Mobile - Footer Menu](https://kanopi.teamwork.com/#/tasks/26124962)
* [Template - Footer - Desktop - Logo & Motto](https://kanopi.teamwork.com/#/tasks/26124963)
* [Template - Footer - Mobile - Motto](https://kanopi.teamwork.com/#/tasks/26124964)
* [Template - Footer - Desktop - Social Media](https://kanopi.teamwork.com/#/tasks/26124965)
* [Template - Footer - Mobile - Social Media](https://kanopi.teamwork.com/#/tasks/26124966)
* [Template - Footer - Desktop - Footer copyright](https://kanopi.teamwork.com/#/tasks/26124967)
* [Template - Footer - Mobile - Footer copyright](https://kanopi.teamwork.com/#/tasks/26124968)
* [Template - Footer - Desktop - Footer Utility Menu](https://kanopi.teamwork.com/#/tasks/26124969)
* [Template - Footer - Mobile - Footer Utility Menu](https://kanopi.teamwork.com/#/tasks/26124970)

## Steps to Validate
1. Visit the [test environment](https://pr-73-rooseveltedu.pantheonsite.io/)
1. Verify the footer looks and responds as required

## Deploy Notes
_Import configurations as normal._

### Testing on local
in the `docksal-local.env` file add/change the `PANTHEON_ENV` value to "pr-73" then run `fin init`. This will get the menu items from the test environment that cannot be added to DEV until the code is merged.
